src/App.jsx

```diff
@@ -15,7 +16,8 @@ function App() {
 
         <Routes>
           <Route path="/" element={<Home />} />
-          <Route path="/article/:article_id" element={<ArticleDetail />} />
+          <Route path="/articles/:article_id" element={<ArticleDetail />} />
+          <Route path="/articles/:article_id/comments" element={<Comments />} />
```

Comment:
From a user experience perspective, would the user want to have to navigate to another page to see the comments or would it be better to have the comments show on the same page as the individual article?